snap/info_snap_yaml.go

```diff
@@ -70,6 +71,7 @@ func InfoFromSnapYaml(yamlData []byte) (*Info, error) {
 		Name:        y.Name,
 		Developer:   y.Developer,
 		Version:     y.Version,
+		Revision:    y.Revision,
```

Comment:
@chipaca I need you here. I have no idea if this change was in your roadmap, or if it doesn't make sense.